.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "assets/Font-Awesome"]
+	path = assets/Font-Awesome
+	url = https://github.com/FortAwesome/Font-Awesome.git
+[submodule "assets/bootstrap-sass"]
+	path = assets/bootstrap-sass
+	url = https://github.com/twbs/bootstrap-sass
```

Comment:
take for example our intern providing https://github.com/os-autoinst/openQA/pull/701 - and I would have to decline this as he forgot to regenerate the cache. 

But possibly we put only in git what we don't already have and generate the rest during package build